cmd/serverless-init/cloudservice/appservice_test.go

```diff
@@ -19,11 +19,24 @@ func TestGetAppServiceTags(t *testing.T) {
 	t.Setenv("APPSVC_RUN_ZIP", "false")
 
 	tags := service.GetTags()
+	tags["aas.environment.os"] = "test_os"
+	tags["aas.environment.runtime"] = "test_runtime"
+	tags["aas.environment.instance_name"] = "test_instance_name"
```

Comment:
Some of these are just blank while others are set by the environment which can differ from local to CI. Is it doing too much if I just check for the existence of the aas keys and not the value?

I feel like down the line, one of the ones that are blank now could have a value later on if the CI environment changes or something